docs/source/syntax/links.md

```diff
@@ -29,7 +29,5 @@ I link to the [Inline link](#inline-link) heading above.
 I link to the [Inline link](#inline-link) heading above.
 
 ```
-I link to the [Notes](tables.html#notes) heading on the [Tables](tables.md) page.
-```
-
-% I link to the [Notes](tables.md#notes) heading on the [Tables](tables.md) page.
```

Comment:
This might be worth a bug report too! would be great to be able to comment lines like this.